src/render/direct3d12/SDL_render_d3d12.c

```diff
@@ -78,6 +84,54 @@
 #define D3D_GUID(X)                   &(X)
 #endif
 
+/*
+ * Older MS Windows SDK headers declare some d3d12 functions with the wrong function prototype.
+ * - ID3D12Heap::GetDesc
+ * - ID3D12Resource::GetDesc
+ * - ID3D12DescriptorHeap::GetDesc
+ * (and 9 more)+
+ * This is fixed in SDKs since WDK_NTDDI_VERSION >= NTDDI_WIN10_FE (0x0A00000A)
+ */
+
+#if !(defined(__MINGW32__) || defined(__XBOXONE__) || defined(__XBOXSERIES__)) \
```

Comment:
@sezero

Reply:
If you saying that WDK_NTDDI_VERSION is checked here and sdkddkver.h is thus needed in all cases, is that not true for the SDL3 tree as well?

Reply:
It is only include when not targeting xbox ==> `!(defined(__XBOXONE__) || defined(__XBOXSERIES__))`.
This test should short circuit the `WDK_NTDDI_VERSION` test.

Reply:
Yes